draft-ietf-taps-interface.md

```diff
@@ -1095,8 +1121,18 @@ The Listen() Action consumes the Preconnection. Once Listen() has been
 called, no further properties may be added to the Preconnection, and no
 subsequent establishment call may be made on the Preconnection.
 
-Listening continues until the global context shuts down, or until the Stop
-action is performed on the same Preconnection:
+~~~
+Connection := Preconnection.ListenPooled()
+~~~
+
+When using Pooled Connections, see {{connection-pooling}}, a pooled
+connection is created instead of a Listener. This connection will receive
+all messages from all underlying transport connections and can be used
+to send replies to them. 
+
+
+In both cases, Listening continues until the global context shuts down,
+or until the Stop action is performed on the same Preconnection:
```

Comment:
... or if I call close on the Connection, in this case?

Reply:
Yepp - added this